src/widgets.rs

```diff
@@ -21,6 +21,7 @@
 //! - [`Tabs`]: displays a tab bar and allows selection.
 //!
 //! [`Canvas`]: crate::widgets::canvas::Canvas
+pub mod ansi_string_buffer;
```

Comment:
I want to mark this as unstable and the widget module and re-export the trait / struct, but this needs https://github.com/sagebind/stability/pull/11 (enable unstable attribute on use statements).